pom.xml

```diff
@@ -105,8 +113,13 @@
                 <configuration>
                     <imageName>nbisweden/ega-mina-inbox</imageName>
                     <baseImage>openjdk:8-jre-alpine</baseImage>
+                    <!-- Next line needs to be a single line, otherwise plugin crashes-->
                     <entryPoint>["java", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005", "-jar", "/${project.build.finalName}.jar", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseCGroupMemoryLimitForHeap"]
                     </entryPoint>
+                    <healthcheck>
+                        <options>--interval=10s</options>
+                        <cmd>curl --fail http://localhost:8080/actuator/health || exit 1</cmd>
```

Comment:
I think we can modify the `health` endpoint address, maybe make it simpler (not using the default one):
```
management.endpoints.web.base-path=/
management.endpoints.web.path-mapping.health=health
```
https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html#production-ready-endpoints-custom-mapping

Reply:
Done.